src/plugins/data/common/data_frames/types.ts

```diff
@@ -105,6 +105,11 @@ export interface IDataFrameResponse extends SearchResponse<any> {
   took: number;
 }
 
+export interface IDataFrameResponseError {
```

Comment:
i think we dont need this anymore right?

Reply:
removed